source/broadband/webpa_adapter.c

```diff
@@ -607,10 +607,27 @@ WDMP_STATUS validate_parameter(param_t *param, int paramCount, REQ_TYPE type)
                         return WDMP_ERR_INVALID_PARAM;
                 }
 
-                if(strlen(param[i].value) >= MAX_PARAMETERVALUE_LEN)
+                //Increase parameter value to 200k for T2 ReportProfiles
+		if(strcmp(param[i].name, "Device.X_RDKCENTRAL-COM_T2.ReportProfiles") == 0) 
                 {
-                        return WDMP_ERR_INVALID_PARAM;
+			WalInfo("----SET/SET-ATTRIBUTES for T2 report profile-----\n");
+			WalInfo("length of parameter value %ld \n",strlen(param[i].value));
+                        if(strlen(param[i].value) >= MAX_TELEMETRY_PARAMETERVALUE_LEN)
+			{
+			WalError("T2 reportProfiles SET/SET-ATTRIBUTES is not supported as string size is more than 200k\n");
+			return WDMP_ERR_INVALID_PARAM;
+			}
                 }
+		else
+		{
+			WalInfo("----SET/SET-ATTRIBUTES for Other than T2-----\n");
+			WalInfo("length of parameter value %ld \n",strlen(param[i].value));
```

Comment:
Same as above, remove this WalInfo logs which is for debugging purpose

Reply:
Incorporated